data/data-pipeline/data_pipeline/tests/sources/dot_travel_composite/test_etl.py

```diff
@@ -0,0 +1,45 @@
+import pathlib
+import geopandas as gpd
+from data_pipeline.tests.sources.example.test_etl import TestETL
+from data_pipeline.etl.sources.dot_travel_composite.etl import (
+    TravelCompositeETL,
+)
+
+
+class TestTravelCompositeETL(TestETL):
+    _ETL_CLASS = TravelCompositeETL
+
+    _SAMPLE_DATA_PATH = pathlib.Path(__file__).parents[0] / "data"
+    _SAMPLE_DATA_FILE_NAME = "DOT_Disadvantage_Layer_Final_April2022.shp"
+    _SAMPLE_DATA_ZIP_FILE_NAME = "Shapefile_and_Metadata.zip"
+    _EXTRACT_TMP_FOLDER_NAME = "TravelCompositeETL"
+
+    def setup_method(self, _method, filename=__file__):
+        """Invoke `setup_method` from Parent, but using the current file name.
+
+        This code can be copied identically between all child classes.
+        """
+        super().setup_method(_method=_method, filename=filename)
+
+    def test_extract_produces_valid_data(self, mock_etl, mock_paths):
+        etl = self._setup_etl_instance_and_run_extract(
+            mock_etl=mock_etl,
+            mock_paths=mock_paths,
+        )
+        df = gpd.read_file(
+            etl.get_tmp_path() / self._SAMPLE_DATA_FILE_NAME,
+            dtype={etl.GEOID_TRACT_FIELD_NAME: str},
+        )
+        assert df.shape[0] == 30
```

Comment:
Idea: we could make expected rows and expected columns into class attributes on the child Test<Name>ETL classes, and then make this test part of the base class? Then we'd have this test coverage everywhere.

Reply:
In general, we do, because we check snapshots of extract for CSVs. In this case, since we're coming from shapefiles, and comparing a shapefile to a CSV didn't feel super useful, I just wanted to check that the file read correctly and was shaped correctly; other later tests verify that we'll get the transformations and ouput we expect.